utils/transcodingFormats.ts

```diff
@@ -0,0 +1,42 @@
+import { browserDetector } from '~/plugins/browserDetection';
+
+/**
+ *
+ */
+export function canPlayNativeHls(): boolean {
+  if (browserDetector.isTizen()) {
+    return true;
+  }
+
+  const media = document.createElement('video');
```

Comment:
Would it be better to pass in the `HTMLVideoElement`?

Reply:
> We probably want to add the few missing JSDOC @returns declaration

I haven't gone through all of these, because there's a ton of functions and they're really annoying to write lol

Reply:
Aha fair enough x)